rails/test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
```

Comment:
Ya dood eff this. What kind of generator are you using? I'm guessing it built all the controllers too. Whatever it is, it's whack.